app/waf/src/controllers/adc.controller.ts

```diff
@@ -424,7 +445,9 @@ export class AdcController extends BaseController {
 
     if (adc.status.endsWith('ING'))
       throw new HttpErrors.UnprocessableEntity(
-        `Adc status is ' ${adc.status}. Cannot be operated on, please wait for its finish.`,
+        `Adc status is ' ${
```

Comment:
You may need to delete `node_modules` folder and `npm install` again. tslint is updated, this line should not have code change.

Reply:
Get it and will following this suggestion accordingly.